## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 515610`
Commit: `6eac42bc64aa627895048dc051e9a0bd4739fbec`

---
### `x86_64-linux`
<details>
  <summary>:x: 7 packages failed to build:</summary>
  <ul>
    <li>python313Packages.zigpy-zboss</li>
    <li>python313Packages.zigpy-zboss.dist</li>
    <li>python314Packages.zigpy-zboss</li>
    <li>python314Packages.zigpy-zboss.dist</li>
    <li>tests.home-assistant-components.otbr</li>
    <li>zigpy-cli</li>
    <li>zigpy-cli.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 53 packages built:</summary>
  <ul>
    <li>home-assistant-custom-components.xiaomi_gateway3</li>
    <li>mew</li>
    <li>python313Packages.bellows</li>
    <li>python313Packages.bellows.dist</li>
    <li>python313Packages.serialx</li>
    <li>python313Packages.serialx.dist</li>
    <li>python313Packages.universal-silabs-flasher</li>
    <li>python313Packages.universal-silabs-flasher.dist</li>
    <li>python313Packages.zha</li>
    <li>python313Packages.zha-quirks</li>
    <li>python313Packages.zha-quirks.dist</li>
    <li>python313Packages.zha.dist</li>
    <li>python313Packages.zigpy</li>
    <li>python313Packages.zigpy-cc</li>
    <li>python313Packages.zigpy-cc.dist</li>
    <li>python313Packages.zigpy-deconz</li>
    <li>python313Packages.zigpy-deconz.dist</li>
    <li>python313Packages.zigpy-xbee</li>
    <li>python313Packages.zigpy-xbee.dist</li>
    <li>python313Packages.zigpy-zigate</li>
    <li>python313Packages.zigpy-zigate.dist</li>
    <li>python313Packages.zigpy-znp</li>
    <li>python313Packages.zigpy-znp.dist</li>
    <li>python313Packages.zigpy.dist</li>
    <li>python314Packages.bellows</li>
    <li>python314Packages.bellows.dist</li>
    <li>python314Packages.serialx</li>
    <li>python314Packages.serialx.dist</li>
    <li>python314Packages.universal-silabs-flasher</li>
    <li>python314Packages.universal-silabs-flasher.dist</li>
    <li>python314Packages.zha</li>
    <li>python314Packages.zha-quirks</li>
    <li>python314Packages.zha-quirks.dist</li>
    <li>python314Packages.zha.dist</li>
    <li>python314Packages.zigpy</li>
    <li>python314Packages.zigpy-cc</li>
    <li>python314Packages.zigpy-cc.dist</li>
    <li>python314Packages.zigpy-deconz</li>
    <li>python314Packages.zigpy-deconz.dist</li>
    <li>python314Packages.zigpy-xbee</li>
    <li>python314Packages.zigpy-xbee.dist</li>
    <li>python314Packages.zigpy-zigate</li>
    <li>python314Packages.zigpy-zigate.dist</li>
    <li>python314Packages.zigpy-znp</li>
    <li>python314Packages.zigpy-znp.dist</li>
    <li>python314Packages.zigpy.dist</li>
    <li>tests.home-assistant-components.hassio</li>
    <li>tests.home-assistant-components.homeassistant_connect_zbt2</li>
    <li>tests.home-assistant-components.homeassistant_green</li>
    <li>tests.home-assistant-components.homeassistant_hardware</li>
    <li>tests.home-assistant-components.homeassistant_sky_connect</li>
    <li>tests.home-assistant-components.homeassistant_yellow</li>
    <li>tests.home-assistant-components.zha</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>python313Packages.zigpy-zboss</summary>
<pre>ERROR tests/api/test_listeners.py::test_api_cancel_listeners - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_cleanup_timeout_internal - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_cleanup_timeout_external - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_zboss_request_kwargs - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_zboss_req_rsp - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_zboss_unknown_frame - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_send_failure_when_disconnected - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_frame_merge - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_responses - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_responses_multiple - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_timeouts - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_matching_partial - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_matching_exact - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_not_matching_out_of_order - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_wait_responses_empty - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_callback_simple - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_callbacks - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_wait_for_responses - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/test_uart.py::test_connection_lost - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
============= 1 failed, 56 passed, 58 warnings, 20 errors in 7.06s =============</pre>
</details>
<details>
<summary>python314Packages.zigpy-zboss</summary>
<pre>ERROR tests/api/test_listeners.py::test_api_cancel_listeners - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_cleanup_timeout_internal - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_cleanup_timeout_external - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_zboss_request_kwargs - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_zboss_req_rsp - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_zboss_unknown_frame - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_send_failure_when_disconnected - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_request.py::test_frame_merge - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_responses - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_responses_multiple - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_timeouts - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_matching_partial - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_matching_exact - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_not_matching_out_of_order - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_wait_responses_empty - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_callback_simple - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_response_callbacks - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/api/test_response.py::test_wait_for_responses - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
ERROR tests/test_uart.py::test_connection_lost - AttributeError: module &#x27;zigpy.serial&#x27; has no attribute &#x27;pyserial_asyncio&#x27;
============ 1 failed, 56 passed, 227 warnings, 20 errors in 6.08s =============</pre>
</details>
<details>
<summary>tests.home-assistant-components.otbr</summary>
<pre>
-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED tests/components/otbr/test_config_flow.py::test_user_flow_additional_entry - AssertionError: No mock registered for GET http://custom_url:1234/api/actio...
FAILED tests/components/otbr/test_config_flow.py::test_user_flow_additional_entry_fail_get_address - AssertionError: No mock registered for GET http://custom_url:1234/api/actio...
FAILED tests/components/otbr/test_config_flow.py::test_user_flow_router_not_setup - AssertionError: No mock registered for GET http://custom_url:1234/api/actio...
FAILED tests/components/otbr/test_config_flow.py::test_user_flow_get_dataset_404 - AssertionError: No mock registered for GET http://custom_url:1234/api/actio...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_yellow - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_sky_connect[/dev/serial/by-id/usb-Nabu_Casa_SkyConnect_v1.0_9e2adbd75b8beb119fe564a0f320645d-if00-port0-Home Assistant Connect ZBT-1 (Silicon Labs Multiprotocol)] - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_sky_connect[/dev/serial/by-id/usb-Nabu_Casa_Home_Assistant_Connect_ZBT-1_9e2adbd75b8beb119fe564a0f320645d-if00-port0-Home Assistant Connect ZBT-1 (Silicon Labs Multiprotocol)] - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_sky_connect[/dev/serial/by-id/usb-Nabu_Casa_ZBT-2_10B41DE58A94-if00-Home Assistant Connect ZBT-2 (Silicon Labs Multiprotocol)] - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_2x_addons - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_2x_addons_same_ext_address - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_router_not_setup - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_router_not_setup_has_preferred - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_router_not_setup_has_preferred_2 - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_flow_404 - AssertionError: No mock registered for GET http://core-silabs-multiprotocol...
FAILED tests/components/otbr/test_config_flow.py::test_hassio_discovery_reload - AssertionError: No mock registered for GET http://core-openthread-border-ro...
================== 16 failed, 84 passed, 2 warnings in 10.16s ==================</pre>
</details>
</details>

--- a/pkgs/development/python-modules/bellows/default.nix
+++ b/pkgs/development/python-modules/bellows/default.nix
@@ -14,14 +14,14 @@
 
 buildPythonPackage rec {
   pname = "bellows";
-  version = "0.49.0";
+  version = "0.49.1";
   pyproject = true;
 
   src = fetchFromGitHub {
     owner = "zigpy";
     repo = "bellows";
     tag = version;
-    hash = "sha256-haWej3ZcUPd9Rpqf2PH8r0useylnLDaPiSctrwLz71Q=";
+    hash = "sha256-dt4cwew/jRpmXaZORfjNCivUMynFbRJITOnmP34Aq+I=";
   };
 
   postPatch = ''

--- a/pkgs/development/python-modules/zha-quirks/default.nix
+++ b/pkgs/development/python-modules/zha-quirks/default.nix
@@ -13,7 +13,7 @@
 
 buildPythonPackage rec {
   pname = "zha-quirks";
-  version = "1.1.1";
+  version = "1.2.0";
   pyproject = true;
 
   disabled = pythonOlder "3.12";
@@ -22,7 +22,7 @@ buildPythonPackage rec {
     owner = "zigpy";
     repo = "zha-device-handlers";
     tag = version;
-    hash = "sha256-GxNxc+cu3wBjz/1VF2+0DJ/PBTLlJKm0ncgzeaw5Fxw=";
+    hash = "sha256-mDcvVwqzSmszaJDahzkRNteiO4C/eU+BqTdBpWj5yGw=";
   };
 
   postPatch = ''

--- a/pkgs/development/python-modules/zha/default.nix
+++ b/pkgs/development/python-modules/zha/default.nix
@@ -23,7 +23,7 @@
 
 buildPythonPackage rec {
   pname = "zha";
-  version = "1.1.2";
+  version = "1.3.0";
   pyproject = true;
 
   disabled = pythonOlder "3.12";
@@ -32,7 +32,7 @@ buildPythonPackage rec {
     owner = "zigpy";
     repo = "zha";
     tag = version;
-    hash = "sha256-GPl3nXi24ukNHDE81keyu8m1xgS0MSRdo7ULxy6foGQ=";
+    hash = "sha256-oB4vxq/DJjmypmcKS6IeYEh+dTvC0Wt9X79vPbtDJgE=";
   };
 
   postPatch = ''
@@ -93,6 +93,7 @@ buildPythonPackage rec {
     "test_startup_concurrency_limit"
     "test_fan_ikea"
     "test_background"
+    "test_gateway_startup_failure" # Failed first attempt, passed second, flaky
   ];
 
   disabledTestPaths = [ "tests/test_cluster_handlers.py" ];

